Common/ML/include/ML/GPUORTFloat16.h

```diff
@@ -0,0 +1,875 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file     GPUORTFloat16.h
+/// \author   Christian Sonnabend <christian.sonnabend@cern.ch>
```

Comment:
Hast du das selbst geschrieben, oder von github kopiert?
In letzten Fall musst du den korrekten Autor angeben, und schauen ob der code unter GPL3 lizensiert ist. Jedenfalls haben wir kein copyright dafür.
Ansonten leg ein 3rdparty Verzeichnis wie `Framework/Foundation/3rdparty` oder `GPU/GPUTracking/display/3rdparty` an. Das wird dann auch im CodeChecker ausgenommen: https://github.com/alisw/alidist/blob/1916f6d88d42959097998d9481b517dc1c1ea84d/o2checkcode.sh#L71

Reply:
Ja, das ist komplett 3rd party von ONNX. Die source geb ich auch im header an, da kopier ich dann vielleicht einfach direkt die file vom ONNX repo und pack die in das Verzeichnis das du oben angegeben hast